graal/src/main/java/io/micronaut/graal/reflect/GraalTypeElementVisitor.java

```diff
@@ -46,7 +46,11 @@
  * @author Iván López
  * @since 1.1
  */
-@SupportedOptions({VisitorContext.MICRONAUT_PROCESSING_PROJECT_DIR})
+@SupportedOptions({
+        VisitorContext.MICRONAUT_PROCESSING_PROJECT_DIR,
+        VisitorContext.MICRONAUT_PROCESSING_GROUP,
+        VisitorContext.MICRONAUT_PROCESSING_MODULE
+})
```

Comment:
Could you move these new declarations to `TypeElementVisitorProcessor` as I believe another processors (like OpenAPI) could potentially using these flags. Thanks.

Reply:
Done